web_src/css/base.css

```diff
@@ -279,6 +279,10 @@ button {
   cursor: pointer;
 }
 
+input {
+  line-height: 100% !important;
```

Comment:
It can't be right if you put a "!important" on a general element/style. It would pollute uncontrollable styles and cause unfixable overriding bugs.

Reply:
What about replacing element selectors with class selectors?

Reply:
Before looking for such workarounds, check why the problem is happening in first place. Does a unstyled `<input>` element reproduce the problem? If not, add the same CSS properties we do until it does show the issue, then you have the culprit.

Reply:
I will test it

Reply:
Likely the problematic style is

```css
.ui.input > input {
  line-height: 1.21428571em;
}
```

Can you reproduce the problem on https://fomantic-ui.com/collections/form.html#form? If you, we can override this style to maybe `1.25em`, just enough to fix your issue basically.

Reply:
![image](https://github.com/go-gitea/gitea/assets/89133723/a96f4d6c-068e-4b26-b9c0-4d691cc224d8)
No ,I can't.

Reply:
Strange, these two inputs should almost be the same styling-wise.

Reply:
I think the only difference is the font we set, so likely your font has too low g-height, and a slight increase to `1.25em` on above style is warranted, so I recommend we put this as a fix into base.css:

```
.ui.input > input {
  line-height: 1.25em; /* fix "g" being cut off */
}
```

Reply:
styles don't seem to work

Reply:
It should work, when I inspect it in devtools, the style from fomantic is active, so can be overwritten with a same selector in base.css for example.

<img width="509" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/62e20c15-4ad7-4216-bb5a-f6f640524092">

Reply:
![image](https://github.com/go-gitea/gitea/assets/89133723/2ddf2721-ebbc-46f2-883c-499b91ad68d7)

Reply:
not found styles of.ui.input > input

Reply:
Scroll down there, it should be somewhere.

Reply:
![image](https://github.com/go-gitea/gitea/assets/89133723/a747b1ea-edec-4526-9e9f-6ff68829ceaf)

Reply:
It should be somewhere, but in any case, I'll probably see what I can do and push a fix here.

Reply:
![image](https://github.com/go-gitea/gitea/assets/89133723/74baee81-ba7e-44e2-b6fe-302394388c27)